src/pudl/metadata/resources/eia861.py

```diff
@@ -627,6 +770,7 @@
         "etl_group": "eia861",
     },
     "core_eia861__yearly_utility_data_nerc": {
```

Comment:
Feels like `core_eia861__yearly_utility_nerc_regions` would be more descriptive as a table name.

Reply:
To be addressed in #3301